docs/install_guides/installing-openvino-from-archive-macos.md

```diff
@@ -30,60 +30,63 @@ You can also check the [Release Notes](https://software.intel.com/en-us/articles
 .. tab:: Software Requirements
 
   * `CMake 3.13 or higher <https://cmake.org/download/>`_ (choose "macOS 10.13 or later"). Add `/Applications/CMake.app/Contents/bin` to path (for default install). 
-  * `Python 3.6 - 3.9 <https://www.python.org/downloads/mac-osx/>`_ (choose 3.6 - 3.10). Install and add to path.
-     * Note that OpenVINO is gradually stopping the support for Python 3.6. Python 3.7 - 3.10 are recommended.
+  * `Python 3.6 - 3.10 <https://www.python.org/downloads/mac-osx/>`_ (choose 3.6 - 3.10). Install and add to path.
+     * Note that OpenVINO is gradually phasing out the support for Python 3.6. Python 3.7 - 3.10 are recommended.
   * Apple Xcode Command Line Tools. In the terminal, run `xcode-select --install` from any directory
   * (Optional) Apple Xcode IDE (not required for OpenVINO™, but useful for development)
 
+.. note::
+   The package may also be installed on other versions of macOS, but only the specific versions listed here are fully validated.
```

Comment:
can we state that we support all macOS versions starting from 10.15 and explicitly mention x86_64 arch?